utils/build/rollup.config.js

```diff
@@ -35,16 +35,16 @@ const externalPackages = [
   'react-uid',
   '@zendeskgarden/react-theming',
   ...Object.keys(pkg.dependencies || {})
-];
+].map(external => new RegExp(`^${external}/?.*`, 'u'));
 
 export default [
   {
     input: pkg['zendeskgarden:src'],
     /**
-     * Only mark common peerDependencies as externals.
-     * These are not included in the bundlesize.
+     * Filter out dependencies that consumers
+     * will bundle during build time
      */
-    external: id => externalPackages.includes(id),
+    external: id => externalPackages.filter(regexp => regexp.test(id)).length > 0,
```

Comment:
Is it valid for the value of `external` to be switched from a boolean return value to a filtered array? Just making sure.

Reply:
Actually we still return a boolean (notice the end of line 47 `....length > 0,`).